### DISCUSSION TITLE
Сортировка треков по длительности

### DISCUSSION THREAD

**Discussion by KisoidPoh095** [Chimildic/YaMuTools#23] [Идея] [answered]
**Сейчас есть:**
По треку (А-Я)
По треку (Я-А)
По исполнителю (А-Я)
По исполнителю (Я-А)
Случайно

**Предлагаю:**
По длительности (короткие-длинные)
По длительности (длинные-короткие)

**Comment by Chimildic** [comment, marked as answer]
Реализовано в версии 0.7.6

**Reply by KisoidPoh095** [reply]
Да, работает. Проверял в Firefox.
